packages/core/src/components/Tooltip/Tooltip.tsx

```diff
@@ -4,7 +4,7 @@ import cx from "classnames";
 import React, { CSSProperties, isValidElement, PureComponent, ReactElement } from "react";
 import { Modifier } from "react-popper";
 import Dialog from "../Dialog/Dialog";
-import { AnimationType, BASE_SIZES_WITH_NONE, HideShowEvent, JustifyType } from "../../constants";
```

Comment:
Maybe we also need to delete the declaration of "BASE_SIZES_WITH_NONE" from the constants?

Reply:
You're right, deleted it